### ISSUE TITLE
Plot for Report Question 1

### ISSUE THREAD

**Issue by GITGoing4321** [uwhpsc-2016/homework4#6]
I'm confused about the plot for Report Question 1. It says that the solutions should all be on the same graph, but it also suggests that all of the solutions should be plotted in the same color, red. I feel like this would make the graph hard to read. I just wanted to confirm that you truly do want them all in red.

**Comment by alyfarahat** [comment]
If you take a look at the plots generated by the tests, you should see that serial and parallel plots have the same color, yet visible because the serial line is think-dashed and the parallel line is continuous.

**Comment by werrah** [comment]
Yes, but for the four different size of N, they all four are supposed to be red. I did both ways -- all in red first, and then one color for each N so it was clearer to me.

**Comment by werrah** [comment]
Yes, but it says: "create a single plot using the example initial condition displaying the results of Nt = 100, 200, 300, 400" and specifies they should all be on the same plot.

You can't tell which curve is for each Nt. They're all red. I did it both ways - one all in red, and one with a different color for each Nt so it was clearer to me.

**Comment by cswiercz** [comment]
For each increasing number of time steps Nt the heat profile will "decay" a bit more. The corresponding solutions should be fairly distinct and slowly approach a constant profile. 

You can plot them in different colors if you like. (@mvelegar) Or, you can try increasing Nt past the suggested number.

**Comment by mvelegar** [comment]
Please feel free to plot using different colors, or change Nt values. It'll be much easier for me to grade clear, well-presented plots!
